### DISCUSSION TITLE
Possible to search by camera model / device model?

### DISCUSSION THREAD

**Discussion by Sea-Worker9552** [xob0t/Google-Photos-Toolkit#26] [General]
I'd like to ilter photos taken on a certain device. Is that possible? I know I can search today on Google Photos using the device name (e.g. iPhone 15 Pro Max) but it would be cool to do batch operations with your tool

**Comment by xob0t** [comment]
Hi!  
This is possible, but would require checking each media item individually, which is slow. The gptk's ui does not provide this functionality, but it is avaliable through its api.  

Alternatively, you can use search as source in GPTK and filter for device that way. This is fast, but GP's search is known to be unreliable, and it could just skip some of the files.  

Good luck!
